core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -120,6 +120,12 @@
     max-width: 200px;
     text-align: center;
   }
+  .popover-content {
```

Comment:
Try creating a unique class name, through this class name you are going to change the styling for all popovers.